File: 스택_큐/[지현] 스택_큐/주식가격.py
def solution(prices):
    answer = []
    for i in range(len(prices) - 1):
        sec = 0
        for j in range(i, len(prices) - 1):
            if(prices[i] <= prices[j]):
                sec += 1
            else:
                break
        answer.append(sec)
    answer.append(0)
    return answer

File: 스택_큐/[지현] 스택_큐/주식가격.js
function solution(prices) {
    var answer = [];
    for (let i = 0; i < prices.length; i++) {
        let count = 1;
        for (let j = i + 1; j < prices.length; j++) {
            if (j != prices.length - 1 && prices[i] <= prices[j]) {
                count++;
            } else {
                answer.push(count);
                break;
            }
        }
    }
    answer.push(0);
    return answer;
}
